Light Vox Engine/Graphics/Camera.cpp

```diff
@@ -9,9 +9,11 @@ Camera::Camera()
     nearZ = 0.01f;
     farZ = 100.f;
 
-    position = glm::vec3( 0.f, 0.f, -2.f );
-    forward = glm::vec3( 0.0f, 0.f, 1.f );
+    position = glm::vec3( 0.f, 0.f, 0.f );
+    forward = glm::vec3( 0.0f, 0.f, -1.f );
     up = glm::vec3( 0.f, 1.f, 0.f );
+    right = glm::vec3( 1.0f, 0.0f, 0.0f );
+    pitchAngle = 0;
 }
 
 Camera::~Camera() {}
```

Comment:
In consistent NameSpace usage here. In the CPP file we should be `using namespace XYZ` and then only have the class type in first part of the definition